tests/e2e/ia-production/ia-prod-base.js

```diff
@@ -0,0 +1,20 @@
+/* global fixture */
+// import { fixture } from 'testcafe';
+import { runBaseTests } from '../helpers/base';
+import BookReader from '../models/BookReader';
+
+const localURL = 'https://archive.org/details/';
+const books = [
+  'birdbookillustra00reedrich', // publicDomain
+  // 'pianoservicingtu00rebl_0', // borrowable,
+  // 'adventuresoftoms00twaiiala' // has chapter markers
```

Comment:
SUGGESTION: Remove commented-out lines (3 above)?

Reply:
Same with this. These items are listed in our BR Dev Guide, and they ultimately should be covered by the end to end tests.  My idea is to pre-load them and as we have time as a community, we will unlock it.